js/modules/timeline.src.js

```diff
@@ -145,7 +145,7 @@ undocumentedSeriesType('timeline', 'line',
              * @default 100
              */
             distance: 100,
-            /* *
+            /**
              * Whether to position data labels alternately. For example, if
              * [distance](#plotOptions.timeline.dataLabels.distance) is set
              * equal to `100`, then the first data label 's distance will be
```

Comment:
Im not sure if it will be clear description for end-user.

Reply:
Suggested description:
Whether to position data labels alternately. For example, if is set equal to `100`, then data labels will be positioned alternately (on both sides of the point) at a distance of 100px.